packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-fundamentals.md

```diff
@@ -0,0 +1,97 @@
+---
+id: About
+section: accessibility
+---
+
+# Accessibility fundamentals 
+
+Accessibility is a foundation of PatternFly, so if you contribute to PatternFly or use it for your products you should first familiarize yourself with our accessibility guidelines and recommendations.
+
+No user should feel left behind when using your product. The goal of accessible design is to remove barriers and create inclusive product experiences that work for everyone, regardless of ability. This is best achieved when accessibility is considered early in the design and development process, but it's never too late to start prioritizing accessibility.
+
+There’s always room to improve accessibility. That’s why our guidelines will continually evolve. Your feedback can help support PatternFly's accessibility, so we welcome you to contribute to our documentation or [get involved in our GitHub discussions](https://github.com/orgs/patternfly/discussions). 
+
+## Assistive technologies
+
+A core consideration for accessibility practices is assistive technologies that people may use to access your product. 
```

Comment:
```suggestion
A core consideration for accessibility practices is the assistive technologies that people may use to access your product. While there are many kinds of assistive technologies, PatternFly focuses on accommodating keyboard and screen reader users, as these are the most common.
```
@thatblindgeye what about something like this?

Reply:
I think that works! Maybe dropping the last bit so it's just "While there are many kinds of assistive technologies, PatternFly focuses primarily on keyboard and screen readers." (or "...accommodating users of keyboards and screen readers.", just for a "user first" approach).